locales/en.json

```diff
@@ -235,7 +233,7 @@
   "systemExcludedFromAnalysis.resumeAnalysis": "Resume vulnerability analysis",
   "systemExcludedFromAnalysis.title": "Excluded from vulnerability analysis",
   "systemKebab.excludeAnalysis": "Exclude system from vulnerability analysis",
-  "systemKebab.resumeAnalysis": "Resume vulnerability analysis for system",
+  "systemKebab.resumeAnalysis": "Include system from vulnerability analysis",
```

Comment:
Should be "Include system in vulnerability analysis", also in mockups there is pluralisation.